backend/test_cluster_analysis.py

```
@@ -0,0 +1,42 @@
+import json
+
+import sys
+import json
```

Ruff (0.11.9)

Redefinition of unused `json` from line 1

Remove definition: `json`

(F811)